core/src/index.ts

```diff
@@ -55,6 +55,8 @@ export { AppsCache } from "./modules/caches/appsCache";
 export { SourcesCache } from "./modules/caches/sourcesCache";
 export { PropertiesCache } from "./modules/caches/propertiesCache";
 
+export { oAuthAccessTokenGetter } from "./modules/oAuth";
```

Comment:
Exporting this means that a plugin wants to use it

Reply:
An alternative approach would be to move this into `plugin.refreshToken('hubspot', oldtoken)` or something like that